pkg/spec/spec.go

```diff
@@ -20,7 +21,7 @@ const Specfile = "spec.json"
 
 // ParseDir parses the given environments `spec.json` into a `v1alpha1.Environment`
 // object with the name set to the directories name
-func ParseDir(baseDir, path string) (*v1alpha1.Environment, error) {
+func ParseDir(baseDir, namespace string) (*v1alpha1.Environment, error) {
```

Comment:
Exactly like that!

It is technically not even required to pass baseDir, as it could be constructed based on `namespace` (better call that entrypoint or path?), but that might be for another refactor

Reply:
PTAL